packages/dart/sshnoports/bundles/core/config/sshnpd.yaml

```diff
@@ -0,0 +1,107 @@
+# Main service's atSign authentication information
+atsign:
+  # The atsign to run the service as
+  # Examples:
+  # `atsign: '@foobar_np'`
+  # `atsign: foobar_np`
+  # Don't do the following:
+  # `atsign: @foobar_np`
+  # The '@' symbol is a special character in yaml.
+  # TODO: MANDATORY SETTING
+  atsign: 
+
+  # The path to the .atKeys file for the atsign, only needed if not
+  # located at the default path.
+  # The default value is dependent on the home directory and the atsign.
+  # Default for the 'atsign' example above:
+  # `keys: ~/.atsign/keys/@foobar_np_key.atKeys`
+  keys: 
+
+  # The passpharase for the atKeys files ("keys" field above) if the file has
+  # been protected with a passphrase.
+  # Example:
+  # `passphrase: 'my-pass-phrase'`
+  passphrase:
+
+  # The root configuration for the atPlatform
+  # Examples:
+  # `root: root.atsign.org` (The default)
+  # `root: proxy:proxy0001.atsign.org:443` (Atsign's proxy for root.atsign.org)
+  # `root: my.enterprise.root:64` (When hosting your own atSigns)
+  # `root: proxy:my.enterprise.proxy:443` (When hosting your own atSigns with a proxy)
+  root: 
+
+# Configuration of access over NoPorts
+# TODO: At least one atSign must be listed under managers or policy
+access:
+  # A single atSign which serves as the policy manager. You must also run a policy
+  # service as that atSign to answer requests.
+  # Example: `policy: foobar03_np`
+  policy:
+
+  # A list of atsigns which serve as "managers" and have direct access to this device
+  # Only permit open permission apply to managers, policy is ignored for managers
+  # Example:
+  # ```
+  # managers:
+  #   - foobar01_np
+  #   - foobar02_np
+  # ```
+  managers:
+    # - your_atsign_here
+
+  # A list of host:port pairs which are allowed to be accessed on this device.
+  # Use policy for finite control by user.
+  # Use "*:*" to represent all host and all ports.
+  # If permit open is left empty, the default value is "localhost:22" and "localhost:3389" for ssh and rdp
```

Comment:
Should we add info regarding behaviour of permit-open when a policy manager is being used - see [this issue](https://github.com/atsign-foundation/noports/issues/1295) for background